src/types.ts

```
@@ -0,0 +1,48 @@
+import type { Bot, DeriveDefinitions, MessageContext, TelegramMessageOrigin, TelegramUser } from 'gramio';
+
+import type { DynamoDBService } from './services/dynamodb.js';
+
+export interface BotConfig {
+    adminGroupId: string;
+    configId: string;
+    setupAt: string;
+    setupBy: TelegramUser;
+}
+
+export type ForwardContext = MessageContext<Bot<{}, DeriveDefinitions>> & {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)